rio/src/RIO/Process.hs

```diff
@@ -157,6 +157,9 @@ type EnvVars = Map Text Text
 
 -- | Context in which to run processes.
 --
+-- On Windows, the context includes filename extensions associated with
```

Comment:
I don't think this comment adds a lot: for non-Windows users it's confusing, and for Windows users it doesn't explain what this means. I'd recommend such a comment be added to any exported identifier allowing the user to override these.

Reply:
OK, I have deleted it. There is no exported identifier allowing the user to override a `ProcessContext`, in this regard. I'll explain what I have done further below.